core/src/com/unciv/models/ruleset/unique/UniqueTriggerActivation.kt

```diff
@@ -121,6 +124,26 @@ object UniqueTriggerActivation {
             else Random(-550) // Very random indeed
         val ruleset = civInfo.gameInfo.ruleset
 
+        // Allow iterating through all unit targets, if needed.
+        if (unique.type?.targetTypes?.contains(UniqueTarget.UnitTriggerable) == true && !unitTriggerableIteration && tile != null &&
+            !UniqueParameterType.UnitTriggerTarget.staticKnownValues.contains(unique.params[0])) {
+            // Every adjacent [mapUnitFilter] unit
+            val triggerFunctions = tile.getTilesInDistance(1) // Adjacent
```

Comment:
tile.neighbors is this, but cached, so faster
Also, doesn't include the current tile which getTilesInDistance does (adjacent to me does not include 'in same tile')

Reply:
Good call, thanks.